### PR TITLE
Simulation switching for omm-server

### PR DESCRIPTION
**Pull request by Ragzouken**

### CODE REVIEW

**Review by rhoslynroebuck — APPROVED** [review]
Yeah this looks good to me. Tested with Subtle Game with no issues.

File: python-libraries/nanover-openmm/src/nanover/openmm/cli.py [hunk 1]
```diff
@@ -22,7 +22,8 @@ def handle_user_arguments() -> argparse.Namespace:
     parser = argparse.ArgumentParser(description=description)
 
     parser.add_argument(
-        "simulation_xml_path",
+        "simulation_xml_paths",
+        nargs="+",
         help="The simulation to run in XML format.",
```
Comment (rhoslynroebuck) [inline python-libraries/nanover-openmm/src/nanover/openmm/cli.py:27]:
```suggestion
        help="The simulations to run in XML format.",
```

File: python-libraries/nanover-openmm/src/nanover/openmm/runner.py [hunk 2]
```diff
@@ -397,3 +433,49 @@ def _register_commands(self):
...
+        self._imd_force = potential_imd_forces[-1]
+
+    def reset(self, app_server, simulation_counter):
+        try:
+            self.simulation.reporters.remove(self.reporter)
+        except ValueError:
+            pass
+        self.reporter = NanoverImdReporter(
+            frame_interval=5,
+            force_interval=10,
```
Comment (rhoslynroebuck) [inline python-libraries/nanover-openmm/src/nanover/openmm/runner.py:472]: Change default to 5 to match rust

Comment (rhoslynroebuck) [inline python-libraries/nanover-openmm/src/nanover/openmm/runner.py:472]:
```suggestion
            force_interval=5,
```

**Comment by Ragzouken** [comment]
> Ran with Subtle Game and got the following issue from the Puppeteering Client:
> 
> ```
> grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
> 	status = StatusCode.UNKNOWN
> 	details = "Exception calling application: list indices must be integers or slices, not float"
> 	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"Exception calling application: list indices must be integers or slices, not float", grpc_status:2, created_time:"2024-06-06T16:40:27.5717304+00:00"}"
> ```

Forgot to make a similar fix to what was on ASE--should be good now, I put in a test to make sure.

**Comment by rhoslynroebuck** [comment]
Tested with Subtle Game again, no problems.

**Pull request merged**